File: Desafio 2.js
var inteiro1 = prompt("Insira o primeiro numero: ")
var inteiro2 = prompt("Insira o segundo numero: ")
var inteiro3 = prompt("Insira o terceiro numero: ")

var maior = inteiro1
if (inteiro2 > maior)
    maior = inteiro2
else if (inteiro3 > maior)
    maior = inteiro3

var menor = inteiro1
if (inteiro2 < menor)
menor = inteiro2
else if (inteiro3 < menor)
menor = inteiro3

alert("O maior numero e " + maior + " e o menor e " + menor)

File: Resoluções_Python/Desafio 2.py
#Faça um programa que leia tres valores inteiros e imprima o maior e o menor deles
valor1 = int(input("Insira o primeiro valor:"))
valor2 = int(input("Insira o segundo valor:"))
valor3 = int(input("Insira o terceiro valor:"))

maiorValor = valor1
if (valor2 > maiorValor):
    maiorValor = valor2
if (valor3 > maiorValor):
    maiorValor = valor3

menorValor = valor1
if (valor2 < menorValor):
    menorValor = valor2
if (valor3 < menorValor):
    menorValor = valor3

print("O maior valor é", maiorValor)
print("O menor valor é", menorValor)